sky/utils/db_utils.py

```diff
@@ -83,4 +106,5 @@ def __init__(self, db_path: str, create_table: Callable):
         # errors. This is a hack, but it works.
         self.conn = sqlite3.connect(db_path, timeout=10)
         self.cursor = self.conn.cursor()
+        enable_wal_mode(self.conn)  # Enable WAL mode
```

Comment:
Instead of having this for all table creation, let's keep it in `creat_table` function as what we did in https://github.com/skypilot-org/skypilot/blob/master/sky/global_user_state.py#L41-L48